draft-ietf-tls-external-psk-importer.md

```diff
@@ -195,7 +203,33 @@ PSKs for TLS 1.2. Indeed, this is necessary for incremental deployment.
 
 # Security Considerations
 
-DISCLAIMER: This is a WIP draft and has not yet seen significant security analysis.
+The Key Importer security goals can be roughly stated as follows: avoid PSK re-use across
+KDFs while properly authenticating endpoints. When modeled as computational extractors, KDFs
+assume that input keying material (IKM) is sampled from some "source" probability distribution
+and that any two IKM values are chosen independently of each other {{Kraw10}}. This
+source-independence requirement implies that the same IKM value cannot be used for two different
+KDFs.
+
+PSK-based authentication is functionally equivalent to session resumption in that a connection
+uses existing key material to authenticate both endpoints. Following the work of
+{{?BAA15=DOI.10.14722/ndss.2015.23277}}, this is a form of compound authentication. Loosely
+speaking, compound authentication is the property that an execution of multiple authentication
+protocols, wherein at least one is uncompromised, jointly authenticates all protocols.
+Authenticating with an externally provisioned PSK, therefore, should ideally authenticate both
+the TLS connection and the external provision process. Typically, the external provision process
+produces a PSK and corresponding context in which the PSK should be used. We refer to an external
```

Comment:
The context is more about where the key came from than where it is going. 
"corresponding context about how the PSK was derived."

Reply:
Hmm... we use `context` for selfie mitigations, which describe where it's going, right? It seems like to can go both ways.